Test-NetStack.psm1

```diff
@@ -907,16 +940,25 @@ Function Test-NetStack {
                         param ( $thisSource, $ClientNodes, $Definitions, $LogFile )
                         Write-Host ":: $([System.DateTime]::Now) :: [Started] N -> Interface $($thisSource.InterfaceIndex) ($($thisSource.IPAddress))"
                         ":: $([System.DateTime]::Now) :: [Started] N -> Interface $($thisTarget.InterfaceIndex) ($($thisSource.IPAddress))" | Out-File $LogFile -Append -Encoding utf8 -Width 2000
+                        $events = @()
 
                         $thisSourceResult = Invoke-NDKPerfNto1 -Server $thisSource -ClientNetwork $ClientNodes -ExpectedTPUT $Definitions.NDKPerf.TPUT
-                       
+
+                        $events = (Get-EventLog System -InstanceId 0x466,0x467,0x469,0x46a)
```

Comment:
I'm wondering if it might be prudent to time box this to last reboot/start of the test? For instance if I run this on my dev box i get results from multiple previous days and I'm not sure that would be helpful. In SDDC we reimage the machines all the time but that might not be the case outside